src/router/router.tsx

```diff
@@ -25,6 +26,11 @@ const Deliveries = lazy(() => import('@/pages/main/Deliveries'))
 const ZonesContent = lazy(() => import('@/pages/main/ZonesContent'))
 
 export default function Router() {
+  const { user } = useAuth();
+
+  const isAuthenticated = !!user;
+  const isAdmin = user?.role === 'admin';
+
   return useRoutes([
```

Comment:
login is still accessible, it shouldnt be if we are logged in.

Reply:
fixed